src/Normalizer/BulkDocsNormalizer.php

```diff
@@ -59,7 +59,7 @@ public function denormalize($data, $class, $format = NULL, array $context = [])
           }
           // Check if the document is a valid Relaxed format.
           elseif (isset($doc['@context'])) {
-            $entity = \Drupal::service('replication.normalizer.content_entity')
+            $entity = \Drupal::service('replication.normalizer.file_entity')
```

Comment:
This was already existing, I know, but this should go through the serializer instead. If your normalizer implements `SerializerAware interface` setSerializer() will get called when the normalizer is added to the serializer service.